docs/sources/flow/tasks/configure/configure-kubernetes.md

```diff
@@ -19,36 +19,56 @@ weight: 200
 
 # Configure {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-To configure {{< param "PRODUCT_NAME" >}} on Kubernetes, perform the following steps:
+This page describes _how_ to apply a new configuration to {{< param "PRODUCT_NAME" >}} 
+when running on Kubernetes with the Helm chart. It assumes that you already have a new configuration that you 
+want to apply to your Helm chart installation. 
 
-1. Download a local copy of [values.yaml][] for the Helm chart.
+If instead you're looking for help in configuring {{< param "PRODUCT_NAME" >}} to perform a specific task,
+for example, to collect and store a specific telemetry signal, check out the [Kubernetes tasks section][k8s-tasks].
```

Comment:
The other pages describe how to apply a very specific agent config to a very specific Helm installation. This page is more generic. As an alternative, we could delete it altogether, but I think we can leave it, we just need to clarify the differences.